### PR TITLE
CVSB-10978

### PR DESCRIPTION
**Pull request by vladddragomir**

### CODE REVIEW

File: src/pages/test-station/test-station-search/test-station-search.ts [hunk 1]
```diff
@@ -13,7 +13,7 @@ import { FirebaseLogsService } from "../../../providers/firebase-logs/firebase-l
 export class TestStationSearchPage implements OnInit {
   @ViewChild('searchBar') searchBar;
   testStations: TestStationReferenceDataModel[] = [];
-  filteredTestStations: TestStationReferenceDataModel[] = [];
+  filteredTestStations: any[] = [];
```
Comment (agoncearenco) [inline src/pages/test-station/test-station-search/test-station-search.ts]: Why did you need to remove the type here?

Comment (vladddragomir) [reply src/pages/test-station/test-station-search/test-station-search.ts]: it was the wrong the type, most probably due to a mistake.
